src/protobuf-net.Test/Issues/Immutables.cs

```diff
@@ -174,6 +194,13 @@ public ImmutableArrayTestClass(ImmutableArray<string> array)
             public ImmutableArray<string> Array {get;}
         }
 
+        [ProtoContract]
+        public class UninitializedImmutableArrayTestClass
+        {
+            [ProtoMember(1)]
+            public ImmutableArray<int> Array { get; }
```

Comment:
Note @mgravell - this test case currently fails, and the fix in this PR fixes it. I believe it's a pretty safe fix and would love to get it merged.